.github/pull-request-template.md

```diff
@@ -0,0 +1,9 @@
+
+**What**  
+- 
+
+**Screen shots**  
+- NA
+
+**Api logs** 
+- NA 
```

Comment:
Maybe we can discuss this in another PR? What kind of PR template we need for the team

Reply:
in same pr we can discuss there is no hurry
i thought to keep simple template by which we can understand whats going on in pr at least 
easy for review.

would you like to change anything from above?

Reply:
Ok since we are on the topic, in my opinion this one is much more detailed and includes guidance for the submitter, but it will depend on the project.

```
### Issue

- close `(required: Put the ticket link)`

### Motivation and Context

- `(required)`

### Modified points

- [ ] `(required: Write "What you did" here)`

### Description and Screenshots

- `(optional)`

### Figma or screen image URL

- `(required)`

### Worries or Issues

- `(optional)`
```

Reply:
Ok since we are on the topic, in my opinion this one is much more detailed and includes guidance for the submitter, but it will depend on the project.

```
### Issue

- close `(required: Put the ticket link)`

### Motivation and Context

- `(required)`

### Modified points

- [ ] `(required: Write "What you did" here)`

### Description and Screenshots

- `(optional)`

### Figma or screen image URL

- `(required)`

### Worries or Issues

- `(optional)`
```